src/librbd/cache/pwl/AbstractWriteLog.cc

```diff
@@ -1463,8 +1463,8 @@ void AbstractWriteLog<I>::alloc_and_dispatch_io_req(C_BlockIORequestT *req)
 template <typename I>
 bool AbstractWriteLog<I>::check_allocation(
     C_BlockIORequestT *req, uint64_t bytes_cached, uint64_t bytes_dirtied,
-    uint64_t bytes_allocated, uint32_t num_lanes, uint32_t num_log_entries,
-    uint32_t num_unpublished_reserves) {
+    uint64_t bytes_allocated, uint64_t num_lanes, uint64_t num_log_entries,
+    uint64_t num_unpublished_reserves) {
```

Comment:
Please open a separate PR for 4G-related changes.

Reply:
> Please open a separate PR for 4G-related changes.

https://github.com/ceph/ceph/pull/41968#discussion_r659358159

Reply:
> Please open a separate PR for 4G-related changes.

https://github.com/ceph/ceph/pull/42046